libavcodec/vvc/inter_template.c

```diff
@@ -483,6 +484,10 @@ static void FUNC(apply_bdof)(uint8_t *_dst, const ptrdiff_t _dst_stride, int16_t
 static void FUNC(dmvr)(int16_t *dst, const uint8_t *_src, const ptrdiff_t _src_stride,
     const int height, const intptr_t mx, const intptr_t my, const int width)
 {
+#if BIT_DEPTH == 10
+    av_image_copy_plane((uint8_t*)dst, sizeof(int16_t) * MAX_PB_SIZE, _src, _src_stride,
```

Comment:
Do you think it will look better if we move it to the end of the function so we can distinguish it from the variable declaration?